### ISSUE TITLE
additional arm configs

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/continuous-integration#21]
@tom-gall might have some more board specific configs we want make sure continue to work.

I'm not sure if we want to get too wide with configs with travis, but maybe this is feasible?

**Comment by nickdesaulniers** [comment]
+ @agners who might have a recommended armv6 config?

**Comment by nathanchance** [comment]
If #20 works out, I don't think it should be too bad.

**Comment by shenki** [comment]
ast2500_g5_defconfig is an armv6 machine (I'm the maintainer for that platform, so I have an interest in seeing it tested).

It has a corresponding qemu machine `-M romulus-bmc` that we can boot.

**Comment by nickdesaulniers** [comment]
@shenki yeah, sounds good then.  We'll need a buildroot image for it.  Have you worked with buildroot before? (I should sit down an write a blog post about it, otherwise I'll soon forget how to do so myself).

**Comment by arndb** [comment]
On 11/3/18, Nick Desaulniers <notifications@github.com> wrote:
> @tom-gall might have some more board specific configs we want make sure
> continue to work.
>
> I'm not sure if we want to get too wide with configs with travis, but maybe
> this is feasible?

multi_v5_defconfig should be small, and covers some of the older platforms.

vf610m4_defconfig or stm32_defconfig would be good examples for ARMv7-M,
I'd suggest the former, since Stefan is familiar with that platform though the
stm32 one is more common.

Adding a big-endian config would also be helpful, as we've had problems
with that in the past.

      Arnd

**Comment by agners** [comment]
@shenki I guess that is the machine you created issue https://github.com/ClangBuiltLinux/linux/issues/90 for?

I guess something with ARMv6 architecture might be interesting anyway. Also RPi1 is ARMv6. There is ARM11MPCore emulation in Qemu, e.g. `realview-eb-mpcore` kernel defconfig `realview_defconfig`.

@arndb yeah I do have hardware to boot vf610m4_defconfig, actually I can boot such a kernel using U-Boot from the Cortex-A core of that SoC. Would be a good candidate for Kernel CI ARMv7-M testing...

**Comment by shenki** [comment]
@agners yes, that's correct

I put up a change that changes the arm32 buildroot to build a generic image (arm926ej-s based, so ARMv5). This way we can share the image for all of the arm32 systems we want to test:

 https://github.com/ClangBuiltLinux/continuous-integration/pull/50
